### ISSUE TITLE
Mobile Audio Jitter

### ISSUE THREAD

**Issue by torch2424** [torch2424/picoDeploy#27]
# Links and Research

### Android Webview Supporting Chrome, therefore better Web Audio API

https://productforums.google.com/forum/#!topic/nexus/Ni35XW6eRR0
https://developer.android.com/about/versions/nougat/android-7.0.html#webview

### Polyfills for Web Audio API

https://github.com/shinnn/AudioContext-Polyfill
https://github.com/jonathantneal/AudioContext

### Articles on Web Audio API

https://www.airpair.com/ionic-framework/posts/using-web-audio-api-for-precision-audio-in-ionic

### Web Audio API Docs

https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API

### Mobile Audio Discussion

https://www.lexaloffle.com/bbs/?tid=30573

**Comment by stickboyc** [comment]
I can't provide any insight here, but I did want to point out that I have a Android 8.1.0 device (Galaxy J3) and am getting the same "jitter" effect. So I don't believe it's isolated to versions below 7. 

Just didn't want anyone chasing the wrong rabbit, since the readme hints that the issue may be related to older Android versions :P

**Comment by torch2424** [comment]
@stickboyc 

Yeah so unfortunately, I think it's an issue with PICO-8 itself. Now that I've written WasmBoy, and know the WebAudioAPI better, it has to be something with how they handle audio in emscripten. So there's nothing I can do about it. You'll notice the same effect if you go on the PICO-8 bbs with your phone.
